Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION := $(shell git describe --tags --abbrev=0)
+VERSION :=v0.5.1.2
```

Comment:
this looks an oversight

Reply:
yeah it was from testing - thanks for the catch

Reply:
fixed